### PR TITLE
Refactor footer component

### PR DESCRIPTION
**Pull request by ankurg22**
In reference to #142

### CODE REVIEW

File: examples/components/footer.hbs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!-- Footer -->
+<footer>
+  <div class="row justify-content-center height-100">
+    <div class="col-8">
+          <div class="row justify-content-center logo-row">
+            <img src="images/online-logo.png" alt="online-logo" class="nav-logo pointer">
+          </div>
+          <div class="d-flex flex-row flex-wrap justify-content-around links">
+            <span class="link">About Us</span>
+            <span class="link">|</span>
```
Comment (aayusharora) [inline examples/components/footer.hbs]: This seems a wrong approach of adding | 
If they exist already, please remove them and use use border-right

File: examples/views/index.hbs [hunk 2]
```diff
@@ -415,7 +415,7 @@
         </div>
     </div>
 
-    
+
```
Comment (aayusharora) [inline examples/views/index.hbs]: Extra spaces, Please remove these spaces

File: examples/components/footer.hbs (hunk 1 above)
Comment (ankurg22) [reply examples/components/footer.hbs]: Done. :)

**Comment by witty123** [comment]
This issue is on hold for now, reopen if we are excepting any prs

**Pull request closed without merging**